src/lib.rs

```diff
@@ -47,9 +43,9 @@ where
     I2C::Error: Into<Error<I2C::Error>>,
 {
     /// Creates a new driver from an I2C peripheral.
-    pub fn new(i2c: I2C) -> Self {
```

Comment:
maybe you can preserve this with the old default value in order not to break the API?

Reply:
```rust
    /// Creates a new driver from an I2C peripheral.
    pub fn new(i2c: I2C) -> Self {
        MCP9808 {
            addr: SlaveAddress::Default.into(),
            i2c,
        }
    }

    /// Change i2c address
    pub fn set_address(&mut self, addr: SlaveAddress) -> u8 {
        self.addr = addr.into();
        self.addr
    }
```
How about this?

Reply:
Sorry been traveling again, looks good! If you increment the crate version I can merge it.